release-tools/prow.sh

```diff
@@ -86,7 +86,7 @@ configvar CSI_PROW_BUILD_PLATFORMS "linux amd64 amd64; linux ppc64le ppc64le -pp
 # which is disabled with GOFLAGS=-mod=vendor).
 configvar GOFLAGS_VENDOR "$( [ -d vendor ] && echo '-mod=vendor' )" "Go flags for using the vendor directory"
 
-configvar CSI_PROW_GO_VERSION_BUILD "1.21.5" "Go version for building the component" # depends on component's source code
+configvar CSI_PROW_GO_VERSION_BUILD "1.22.2" "Go version for building the component" # depends on component's source code
```

Comment:
Go 1.22.3 was released on 5/7 [1]. Is it possible to include that update in this release?
Note: updating should resolve the Trivy gate failure, too.

[1] https://groups.google.com/g/golang-announce/c/wkkO4P9stm0

Reply:
Regarding this matter, I believe it will be necessary to update the csi-release-tools submodule in this repository after merging the following PR:
kubernetes-csi/csi-release-tools#252